proto/projects/project.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package lavanet.lava.projects;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/lavanet/lava/x/projects/types";
+
+message Project {
+    string index = 1; // unique id that will be the combination of subscription address and project name, cannot be changed once created
+    string subscription = 2; // the subscription address that owns the project
```

Comment:
why not "owner"? it's clearer

Reply:
because it is the address of the subscription, owner feels vague